src/loaders/HttpError.js

```diff
@@ -0,0 +1,12 @@
+class HttpError extends Error {
```

Comment:
Do you mind moving this custom error class into `FileLoader` instead?

Reply:
Sure thing